mitmproxy/flow_export.py

```diff
@@ -1,7 +1,25 @@
 import urllib
-import netlib.http
 from textwrap import dedent
 
+import netlib.http
+from . import contentviews
+
+
+def prettify(data, headers=None):
+    if not headers:
+        return data
+
+    cv = contentviews.get_content_view(
+        contentviews.get("Auto"),
+        data,
+        headers=headers,
+    )
+
+    if cv[0] == "JSON":
+        return "\n".join(l[0][1] for l in cv[1])
```

Comment:
Currently, I am just hardcoding this for JSON only as I couldn't figure out a way to make it work for all the cases. Help?